packages/video_player_avplayer/tizen/src/plus_player.cc

```diff
@@ -0,0 +1,608 @@
+  switch (track_type) {
+    case 1:
+      type = plusplayer::TrackType::kTrackTypeAudio;
+      break;
+    case 2:
+      type = plusplayer::TrackType::kTrackTypeVideo;
+      break;
+    case 3:
+      type = plusplayer::TrackType::kTrackTypeSubtitle;
+      break;
+    default:
+      LOG_ERROR("[PlusPlayer] Invalid track type: %d", track_type);
+      return {};
+  }
+  if (!player_->SelectTrack(type, track_id)) {
+    LOG_ERROR("[PlusPlayer] Player select track failed.");
+    return false;
+  }
+  return true;
+}
+
+bool PlusPlayer::SetDrm(const std::string &uri, int drm_type,
+                        const std::string &license_server_url) {
+  drm_manager_ = std::make_unique<DrmManager>();
+  if (!drm_manager_->CreateDrmSession(drm_type, true)) {
+    LOG_ERROR("[PlusPlayer] Failed to create drm session.");
+    return false;
+  }
+
+  int drm_handle = 0;
+  if (!drm_manager_->GetDrmHandle(&drm_handle)) {
+    LOG_ERROR("[PlusPlayer] Failed to get drm handle.");
+    return false;
+  }
+
+  plusplayer::drm::Type type;
+  switch (drm_type) {
+    case DrmManager::DrmType::DRM_TYPE_PLAYREADAY:
+      type = plusplayer::drm::Type::kPlayready;
+      break;
+    case DrmManager::DrmType::DRM_TYPE_WIDEVINECDM:
+      type = plusplayer::drm::Type::kWidevineCdm;
+      break;
+    default:
+      type = plusplayer::drm::Type::kNone;
+      break;
+  }
+
+  plusplayer::drm::Property property;
+  property.handle = drm_handle;
+  property.type = type;
+  property.license_acquired_cb =
+      reinterpret_cast<plusplayer::drm::LicenseAcquiredCb>(OnLicenseAcquired);
+  property.license_acquired_userdata =
+      reinterpret_cast<plusplayer::drm::UserData>(this);
+  property.external_decryption = false;
+  player_->SetDrm(property);
+
+  if (license_server_url.empty()) {
+    bool success = drm_manager_->SetChallenge(
+        uri, [this](const void *challenge, unsigned long challenge_len,
+                    void **response, unsigned long *response_len) {
+          OnLicenseChallenge(challenge, challenge_len, response, response_len);
+        });
+    if (!success) {
+      LOG_ERROR("[PlusPlayer]Failed to set challenge.");
+      return false;
+    }
+  } else {
+    if (!drm_manager_->SetChallenge(uri, license_server_url)) {
+      LOG_ERROR("[PlusPlayer]Failed to set challenge.");
+      return false;
+    }
+  }
+  return true;
+}
+
+bool PlusPlayer::OnLicenseAcquired(int *drm_handle, unsigned int length,
+                                   unsigned char *pssh_data, void *user_data) {
+  LOG_INFO("[PlusPlayer] License acquired.");
+
+  PlusPlayer *self = static_cast<PlusPlayer *>(user_data);
+  if (self->drm_manager_) {
+    return self->drm_manager_->SecurityInitCompleteCB(drm_handle, length,
+                                                      pssh_data, nullptr);
+  }
+  return false;
+}
+
+void PlusPlayer::OnPrepareDone(bool ret, UserData userdata) {
+  LOG_DEBUG("[PlusPlayer] Prepare done, result: %d.", ret);
```

Comment:
Only this log is LOG_DEBUG. I think it should be LOG_INFO

Reply:
Fixed.